src/cdi_CPEloss/test/tstAnalytic_CP_Eloss.cc

```diff
@@ -107,9 +108,88 @@ void KP_alpha_test(rtt_dsxx::UnitTest &ut) {
   }
 
   if (ut.numFails == 0)
-    PASSMSG("Analytic_CP_Eloss test passes.");
+    PASSMSG("KP_Alpha CPEloss test passes.");
   else
-    FAILMSG("Analytic_CP_Eloss test fails.");
+    FAILMSG("KP_Alpha CPEloss test fails.");
+
+  return;
+}
+
+void Spitzer_test(rtt_dsxx::UnitTest &ut) {
+
+  // Deuterium:
+  int deuterium_zaid = 1002;
```

Comment:
Even though this is just a test, it might be good to explicitly use int32_t here to match the signature!